frontend/src/components/SecretRow.vue

```diff
@@ -5,112 +5,109 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 <template>
-  <v-list-item>
-    <v-list-item-content>
-      <v-list-item-title class="mb-1">
-        {{secret.metadata.name}}
-        <v-tooltip v-if="!isOwnSecret" top>
+  <tr>
+    <td v-if="selectedHeaders.name">
+      <div class="d-flex">
+        {{item.name}}
+        <v-tooltip v-if="!item.isOwnSecret" top>
           <template v-slot:activator="{ on }">
             <v-icon v-on="on" small class="mx-1">mdi-account-arrow-left</v-icon>
           </template>
-          <span>Secret shared by {{secretOwner}}</span>
+          <span>Secret shared by {{item.ownernamespace}}</span>
         </v-tooltip>
-        <span style="opacity:0.5">({{relatedShootCountLabel}})</span>
-      </v-list-item-title>
-      <v-list-item-subtitle>
-        <slot name="rowSubTitle" :data="secret.data">{{secretDescriptor}}</slot>
-      </v-list-item-subtitle>
-    </v-list-item-content>
-
-    <v-list-item-action>
+      </div>
+    </td>
+    <td v-if="selectedHeaders.infrastructure">
       <v-tooltip top>
         <template v-slot:activator="{ on }">
-          <div v-on="on">
-            <v-btn :disabled="isDeleteButtonDisabled" icon @click.native.stop="onDelete">
-              <v-icon class="error--text">mdi-delete</v-icon>
-            </v-btn>
+          <div class="d-flex align-center" v-on="on">
+            <infra-icon v-on="on" v-model="item.infrastructure"></infra-icon>
+            <span class="ml-2">{{item.cloudProfileName}}</span>
           </div>
         </template>
-        <span v-if="!isOwnSecret">You can only delete secrets that are owned by you</span>
-        <span v-else-if="relatedShootCount > 0">You can only delete secrets that are currently unused</span>
-        <span v-else>Delete Secret</span>
+        <div><span class="font-weight-bold">Infrastructure:</span> {{item.infrastructure}}</div>
+        <div><span class="font-weight-bold">Cloud Profile:</span> {{item.cloudProfileName}}</div>
       </v-tooltip>
```

Comment:
align tooltip with the one on the shoot list

Reply:
Do you mean that I should use the Vendor component? Yes, that was my first idea but this tooltip shows shoot information (zone, region) .. so this does not possible (out of the box). It would be an option to pass the cloud information as parameters and not the shootItem but I don't know if reuse makes sense here as we show completely different information (`Infrastructure / Cloud Profile` vs `Infrastructure / Region / Zone`)

Reply:
No I was not saying to (re)use the component, just to mimic the same visual style

Reply:
Hmm. But then we need to rethink this in general. I had a discussion with @holgerkoser a few weeks ago about the visual style of this tooltip does not match the other ones (white background).

Reply:
I changed it to white, otherwise the icons would not be nicely visible, but meanwhile we show a box around icons so I guess we can at least change the background

Reply:
switched back to white and re-use on secrets list